parm/metplus_config/mesoscale/grid2obs/stats/PointStat_fcstMESOSCALE_obsMETAR.conf

```diff
@@ -28,7 +28,7 @@ POINT_STAT_VERIFICATION_MASK_TEMPLATE = {ENV[MASK_CUSTOM_LIST]}
 LOG_LEVEL = {ENV[METPLUS_VERBOSITY]}
 LOG_MET_VERBOSITY = {ENV[MET_VERBOSITY]}
 LOG_POINT_STAT_VERBOSITY = {LOG_MET_VERBOSITY}
-LOG_MET_OUTPUT_TO_METPLUS = {ENV[LOG_MET_OUTPUT_TO_METPLUS]}
+#LOG_MET_OUTPUT_TO_METPLUS = {ENV[LOG_MET_OUTPUT_TO_METPLUS]}
```

Comment:
@RoshanShrestha-NOAA , what do you think about removing this commented out line from all of your parm files? We have demonstrated that not having it is OK now.